setup.py

```diff
@@ -1,11 +1,31 @@
 # Always prefer setuptools over distutils
+
+
+import os
 from setuptools import setup, find_packages
 
 
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+with open(os.path.join(here, 'VERSION')) as f:
+    __version__ = f.read().strip()
+
+
+with open(os.path.join(here, 'requirements.txt')) as f:
```

Comment:
Do we even need to keep `requirements.txt` around? If we can I'd just get rid of it. Everyone who installs via `pip instal osfclient` will get what is specified in `setup.py`. If `pip install -e.` also installs those dependencies then even people who clone the GitHub repo will be taken care of. Not sure if this works though.

Reply:
gosh, that's an interesting idea 🤔

personally I would keep it, mostly because I'm used to using it for my workflow

in addition to pip, you can spawn a conda environment from requirements.txt `conda create -n myenvname --file requirements.txt`

in the back of my mind a little voice is saying that readthedocs wants requirements.txt

I like having little text files that you can view quickly, copy and paste etc so requirements.txt makes sense to me

that said I suspect you are correct---we could get rid of the file

up to you @betatim, I'm happy to remove

Reply:
Ah yes, I had forgotten about RTD.